man/systemd.exec.xml

```diff
@@ -440,6 +440,14 @@
         <citerefentry project='freebsd'><refentrytitle>inetd</refentrytitle><manvolnum>8</manvolnum></citerefentry>
         daemon.</para>
 
+        <para>The <option>named-descriptor</option> option connects
+        the input stream to a single file-descriptor provided by a socket unit.
+        This option requires a descriptor named <literal>StandardInput</literal>;
+        if multiple named matches are found, the first one will be used.
+        See <varname>FileDescriptorName=</varname> in
+        <citerefentry><refentrytitle>systemd.socket</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+        for more details about named descriptors and ordering.</para>
+
```

Comment:
You are right, I missed the initial list in the doc. I'll add it in the next iteration.